src/Storages/CompressionCodecSelector.h

```diff
@@ -45,6 +45,7 @@ class CompressionCodecSelector
     {
         size_t min_part_size = 0;
         double min_part_size_ratio = 0;
+        bool enable_deflate_qpl_codec = false;
```

Comment:
Let's add a tag for enable_deflate_qpl_codec in l. 30.

Reply:
Done